assets/styles/_variables.scss

```diff
@@ -15,150 +15,29 @@ $aspect-ratios: (
 ) !default;
 
 :root {
-  --kbin-body-font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```

Comment:
I think you should keep the common settings/CSS variables in this `assets/styles/_variables.scss`.

Reply:
yea I see, you are saying this is causing this flash issues?